sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -194,16 +193,11 @@ def _create_tasks_for_claimed_ownership(
         self, partition_context: PartitionContext, event: EventData
     ) -> None:
         with self._context(event):
-            try:
-                if self._track_last_enqueued_event_properties:
-                    partition_context._last_received_event = (  # pylint: disable=protected-access
-                        event
-                    )
-                await self._event_handler(partition_context, event)
-            except asyncio.CancelledError:  # pylint: disable=try-except-raise
-                raise
-            except Exception as error:  # pylint:disable=broad-except
-                await self._process_error(partition_context, error)
+            if self._track_last_enqueued_event_properties:
+                partition_context._last_received_event = (  # pylint: disable=protected-access
+                    event
+                )
+            await self._event_handler(partition_context, event)
```

Comment:
Shoud it call process_error when the exception is not CancelledError?

Reply:
process_error would be called in the loop of the `_receive`